src/homekit.cs

```diff
@@ -1671,12 +1683,63 @@ interface HMNumberRange {
 		NSNumber Max { get; }
 	}
 
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAccessoryOwnershipToken {
+		[Export("initWithData:")]
+		IntPtr Constructor (NSData data);
+	}
+
+	[iOS (13,0), NoWatch, NoMac, NoTV]
+	[DisableDefaultCtor]
+	interface HMAddAccessoryRequest {
+		[Export ("home")] 
+		HMHome Home { get; }
+		
+		[Export ("accessoryCategory")]
+		HMAccessoryCategory AccessoryCategory { get; }
+		
+		[Export ("accessoryName")]
+		NSString AccessoryName { get; }
+		
+		[Export ("requiresSetupPayloadURL")]
+		bool RequiresSetupPayloadUrl { get; }
+		
+		[Export ("requiresOwnershipToken")]
+		bool RequiresOwnershipToken { get; }
+
+		[NullAllowed, Export ("payloadWithOwnershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithOwnershipToken (HMAccessoryOwnershipToken ownershipToken);
+		
+		[NullAllowed, Export ("payloadWithURL:ownershipToken:")]
+		HMAccessorySetupPayload GetPayloadWithUrlAndOwnershipToken (NSUrl setupPayloadUrl, HMAccessoryOwnershipToken ownershipToken);
```

Comment:
```suggestion
		HMAccessorySetupPayload GetPayload (NSUrl setupPayloadUrl, HMAccessoryOwnershipToken ownershipToken);
```

This is the Objective-C naming, in C# it's inferred from the parameter names (: